fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>subrat&ridwan</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  <div id="container">
+    <div id="header">
+       <h1 id="title">freeCodeCamp Survey Form Survey Form</h1>
+       <p id="description"><em>Thank you for taking the time to help us improve the platform</em></p>
+    </div>
```

Comment:
- [ ] I'd suggest this section is wrapped in a `header` tag😉 as it makes the page more semantic and readable👍

Reply:
Great job asking for changes politely say the why and using emoji's😊, keep it up👍.